### ISSUE TITLE
Multi-Aperture Photometry

### ISSUE THREAD

**Issue by kjkoeller** [viveikjha/aperture_photometry#3]
Hello,

I recently graduated with a masters degree and my thesis was on W UMa binary stars and I created a package for analyzing these systems. I am working on creating a pipeline for Ball State University's observatory but for a really good pipeline I need a working multi-aperture photometry program. I found your package while searching GitHub and I was wondering if you all had any intentions of making a working multi-aperture photometry program or the knowledge of making that. 

My current, non-working, version is here (https://github.com/kjkoeller/EclipsingBinaries/blob/main/EclipsingBinaries/multi_aperture_photometry.py)

**Comment by viveikjha** [comment]
Hi,

Sorry for the late reply. Yes, I indeed plan to make it a flexible and workable aperture photometry package. In fact, one of the students successfully generated light curves for a few hundred sources using this code last month only. The only problem I see is that the code is not robust at the moment, and there are a few problems which need to be addressed manually. It'd be great if we could streamline it. 'final_notebook.py' is the latest version of the code, which performs all the steps in a single script. Please let me know if you face any problems using it.

**Comment by kjkoeller** [comment]
Have you thought about incorporating comparison stars to find the magnitude rather than just using the relative flux?

**Comment by viveikjha** [comment]
Yes, I have tried that too, but the problem is that we won't have comparison stars in every frame. Another way would be to search for my sources on simbad and get the magnitudes for matching sources from there. We can then compare and see if there is a constant offset. It is working fine for most of the frames I have right now.

**Comment by kjkoeller** [comment]
When you did attempt to use comparison stats, did you get the magnitude calculation to be accurate?
